### DISCUSSION TITLE
Ookla servers selection

### DISCUSSION THREAD

**Discussion by Cradlepoint01** [alexjustesen/speedtest-tracker#1061] [Q&A]
Hi, could somebody help me with the steps to install this app? I'm interested to do the speed tests (consecutive speed tests) using the same Ookla server to have consistent results. The Cradlepoint speed test isn't consistent, the Ookla servers are random choose and the results are wrong. What I understood reading the document, in this app the last 20 closest servers are shown and can be selected for tests which is great. Please help me with the steps for installation, I really want to test this app. Kind regards.
P.S. it's possible to see the results together with the GPS position?

**Comment by alexjustesen** [comment]
GPS isn't possible as the application has no context of your geo location, just the host machines IP address. 

You could use something like https://williamyaps.github.io/wlmjavascript/servercli.html or https://sparanoid.com/lab/speedtest-list/ to find servers out of your range. 

If you need help setting up the application I recommend having a basic understanding of Docker, Docker Compose and being comfortable setting up host machine to run it on.

**Comment by Cradlepoint01** [comment]
Hi,Thanks for explanations. My problem is that I cannot set an Ookla server in the Cradlepoint application. The server is chosen randomly by the application. I want to use the same server for all speed test sessions in an area, when you use the same server the throughput is consistent otherwise you can't use them for statistics. I will try to "decrypt" the Cradlepoint setting for docker.
Kind regards

**Reply by alexjustesen** [reply]
I'm not familiar with Cradle point unfortunately

**Comment by edzhelyov** [comment]
I'm note sure, but how can you update the server list?

**Comment by Cradlepoint01** [comment]
Hi, probably using the Ookla server ID, name or ip.Br
    On Thursday, January 18, 2024 at 10:30:46 PM GMT+2, Evgeni Dzhelyov ***@***.***> wrote:  
 
 


I'm note sure, but how can you update the server list?

**Comment by edzhelyov** [comment]
Sorry, I wasn't very detailed in my question.

I want to have a predefined list of servers to check each time. At the moment I have a list of 20 local servers, but I would like to import my own list of Ookla servers and if possible to select which ones to check.

Is that possible with the current speedtest-tracker setup?
I couldn't find anything about that in the documentation.

**Comment by alexjustesen** [comment]
Adding https://github.com/alexjustesen/speedtest-tracker/discussions/245 for lineage, closing as duplicate.
